dashboard/app/helpers/application_helper.rb

```diff
@@ -13,6 +13,8 @@ module ApplicationHelper
 
   USER_AGENT_PARSER = UserAgentParser::Parser.new
 
+  PUZZLE_PAGE_NONE = -1
```

Comment:
Nit: can this be scoped down?  `ScriptLevelsHelper` or `LevelsHelper` may be good options.